CONTRIBUTING.md

```diff
@@ -37,7 +24,7 @@ Read the READMEs in [`android/`](android/README.md) and [`ios/`](ios/README.md)
 
 ### iOS
 
-1. Open the `react-native-mediapipe/example/ios/MediapipeExample.xcworkspace` file with Xcode
+1. Open the `react-native-mediapipe/examples/ios/MediapipeExample.xcworkspace` file with Xcode
```

Comment:
That is still not the correct path - there is a folder called objectdetection under /examples/ and the xcworkspace file is called differently.